hrm-domain/lambdas/files-urls/getSignedS3Url.ts

```diff
@@ -51,8 +51,17 @@ const getSignedS3Url = async (event: AlbHandlerEvent): Promise<GetSignedS3UrlRes
     return parseParametersResult;
   }
 
-  const { accountSid, bucket, key, method, objectType, objectId, fileType } =
-    parseParametersResult.data;
+  const {
+    accountSid,
+    bucket,
+    key: encodedKey,
+    method,
+    objectType,
+    objectId,
+    fileType,
+  } = parseParametersResult.data;
+
+  const key = decodeURIComponent(encodedKey);
```

Comment:
would it make sense to move this into the parameter parser instead since that is responsible for taking the arguments provided to the lambda and validating/parsing them for use by the rest of the system?